### PR TITLE
Adding mass to dynamics

### PR DESCRIPTION
**Pull request by dfridovi**
Pretty self-explanatory. Shouldn't break anything. Nice little option to generate stochastic dynamics.

### CODE REVIEW

File: src/dynamics.jl [hunk 1]
```diff
@@ -9,17 +9,18 @@ function planar_double_integrator(; dt = 0.1, kwargs...)
             0.0 0.0 0.0 1.0
         ],
         B = [
-            dt2 0.0
-            0.0 dt2
-            dt 0.0
-            0.0 dt
+            dt2    0.0
```
Comment (lassepe) [inline src/dynamics.jl]: Isn't there an `m` missing here as well?

Comment (dfridovi) [reply src/dynamics.jl]: oof wait how did it not get that change? git...

Comment (dfridovi) [reply src/dynamics.jl]: wait no I can't read. yes this is correct as written. states are `position, velocity` so the upper block should not include `mass`. all I did was change the input to `force` from `acceleration`

**Review by lassepe — APPROVED** [review]
lgtm

**Pull request merged**